### ISSUE TITLE
Additional vehicle methods/properties

### ISSUE THREAD

**Issue by jovanivanovic** [altmp/altv-issues#592]
By recommendation of @7Hazard I am writing a list of methods/properties to be added to the Vehicle class on the server if possible, if not then just on the client.

```cpp
steeringAngleOffset = (int)(addr + 6) + 8
steeringAngleInputOffset = (int)(addr + 6)

throttlePOffset = (int)(addr + 6) + 0x10
throttleOffset = (int)(addr + 10) + 0x10 // idk the difference between this and throttlePOffset

brakePOffset = (int)(addr + 6) + 0x14
handbrakeOffset = (int)(addr + 3) // exists clientside as native but might be a nice addition?

turboOffset = (int)(addr + 4)

wheelSteeringAngleOffset = (int)(addr + 3)
wheelBrakeOffset = ((int)(addr + 3)) + 0x4

clutchOffset = (int)(addr + 10) + 0xC
currentGearOffset = (int)(addr + 3) + 2 // getter for gear exists on clientside, but no setter
nextGearOffset = (int)(addr + 3)
```

And also couple of FiveM-specific native functions:
- [GET_VEHICLE_INDICATOR_LIGHTS](https://github.com/citizenfx/fivem/blob/abd36a905035fb0779c83fdb0c887fab4af76a00/code/components/extra-natives-five/src/VehicleExtraNatives.cpp#L714)
- [GET_VEHICLE_STEERING_SCALE](https://github.com/citizenfx/fivem/blob/abd36a905035fb0779c83fdb0c887fab4af76a00/code/components/extra-natives-five/src/VehicleExtraNatives.cpp#L664)
- [SET_VEHICLE_STEERING_SCALE](https://github.com/citizenfx/fivem/blob/abd36a905035fb0779c83fdb0c887fab4af76a00/code/components/extra-natives-five/src/VehicleExtraNatives.cpp#L665)

**Comment by Artiik373** [comment]
+1 for this best shit bro

**Comment by DurtyFree** [comment]
I think I covered almost all additional properties in my list: https://github.com/altmp/altv-issues/issues/355

**Comment by Artiik373** [comment]
> I think I covered almost all additional properties in my list: #355

guess this can be closed as dupe of #355 then

**Comment by Lhoerion** [comment]
Duplicate of #355
